src/comp/comp.py

```diff
@@ -21,51 +21,55 @@ def __repr__(self):
     Human("David", 31),
 ]
 
+print(humans[0].name)
+print(type(humans[0].age))
+
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+
+a = [name.name for name in humans if name.name[0] == "D"]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [name.name for name in humans if name.name[-1] == "e"]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+c = [name.name for name in humans if name.name[0] == "C" or name.name[0] == "D" or name.name[0] == "E" or name.name[0] == "F" or name.name[0] == "G"]
```

Comment:
Perfectly valid solution that demonstrates a clear understanding of how list-comprehensions work.

One way to simplify this is to know that strings are iterable sequences just like lists, tuples, dictionaries and sets. This allows us to take advantage of the `in` keyword.

With this in mind, instead of checking for an equality between two characters, we can check if a character is `in` a sequence of characters (or a string).

This allows us to reduce:

```
if name.name[0] == "C" or name.name[0] == "D" or name.name[0] == "E" or name.name[0] == "F" or name.name[0] == "G"
```

to

```
if name.name[0] in "CDEFG"
```

and therefore

```
c = [name.name for name in humans if name.name[0] in "CDEFG"]
```

Your solution is perfectly fine though and works just as well, really nice job 🎉